packages/ckeditor5-table/tests/commands/insertcolumncommand.js

```diff
@@ -5,28 +5,27 @@
 
 import ModelTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/modeltesteditor';
 import HorizontalLineEditing from '@ckeditor/ckeditor5-horizontal-line/src/horizontallineediting';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
 import { getData, setData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
+import { assertEqualMarkup } from '@ckeditor/ckeditor5-utils/tests/_utils/utils';
 
-import InsertColumnCommand from '../../src/commands/insertcolumncommand';
 import TableSelection from '../../src/tableselection';
-import { assertSelectedCells, defaultConversion, defaultSchema, modelTable } from '../_utils/utils';
-import TableUtils from '../../src/tableutils';
-import { assertEqualMarkup } from '@ckeditor/ckeditor5-utils/tests/_utils/utils';
+import TableCellPropertiesEditing from '../../src/tablecellproperties/tablecellpropertiesediting';
+import { assertSelectedCells, modelTable } from '../_utils/utils';
+
+import InsertColumnCommand from '../../src/commands/insertcolumncommand';
 
 describe( 'InsertColumnCommand', () => {
 	let editor, model, command;
 
 	beforeEach( () => {
 		return ModelTestEditor
 			.create( {
-				plugins: [ TableUtils, TableSelection, HorizontalLineEditing ]
+				plugins: [ Paragraph, TableCellPropertiesEditing, TableSelection, HorizontalLineEditing ]
```

Comment:
Why `TableCellPropertiesEditing` is used here instead of `TableEditing` ?

Reply:
Looks like typo :D